src/generator/generator.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+#include <vector>
+#include <memory>
+#include "src/algorithms/algorithm.h"
+
+class Generator
```

Comment:
Generator of what? We will have random number generator etc.

Reply:
I'll change that to ProceduralContentGenerator

Reply:
Great, Jakub